nbs/API/migrate.ipynb

```diff
@@ -1,23 +1,22 @@
 {
```

Comment:
<p>I am not longer using <code>nbdev.clean.process_write</code> I suspect that particular thing could be refactored in a follow on PR I found it easier to use Processors </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>